addons/purchase/views/res_config_views.xml

```diff
@@ -41,7 +41,7 @@
                                 <div class="o_setting_right_pane">
                                     <label for="module_purchase_requisition"/>
                                     <div class="text-muted">
-                                        Get the best offer from several vendors
+                                        Manage your purchase agreements (call for tenders, blanket orders, etc.)
```

Comment:
Remove the etc.

Manage your purchase agreements (call for tenders, blanket orders).